doc/api/fs.markdown

```diff
@@ -822,13 +822,20 @@ Returns a new WriteStream object (See `Writable Stream`).
     { flags: 'w',
       encoding: null,
       fd: null,
-      mode: 0666 }
+      mode: 0666,
+      autoClose: true }
 
 `options` may also include a `start` option to allow writing data at
 some position past the beginning of the file.  Modifying a file rather
 than replacing it may require a `flags` mode of `r+` rather than the
 default mode `w`.
 
+If `autoClose` is false, then the file descriptor won't be closed, even if
+there's an error. It is your responsiblity to close it and make sure
```

Comment:
@saquibkhan Thank you very much for your first contribution!

There's no rush to merge a PR. Taking the extra time to make sure changes are as good as they can be and well understood always pays off. We don't want to rush merging changes and introduce new problems that will take more time to fix.

In this case however I'm confident we'll be able to merge it very soon, so there's no need to worry :)